src/network/mcpe/convert/TypeConverter.php

```diff
@@ -56,6 +56,10 @@
 class TypeConverter{
 	use SingletonTrait;
 
+	//TODO: Hack! In Bedrock smithing recipe indexes go together with regular recipes,
+	//so we make an offset to prevent conflicts.
+	public const SMITHING_RECIPES_NETWORK_OFFSET = 200000;
```

Comment:
It would be better if this was declared together with the other offset constants so that it's clear which ones have already been used. (That being said, the 100k for enchanting options is currently hardcoded in `InventoryManager`, so make of that what you will.)